dev/cognitiveservices/data-plane/Language/analyzeconversations.json

```diff
@@ -2042,6 +2042,35 @@
           "type": "string",
           "description": "For transcript conversations, this parameter provides information regarding which content type (ITN, Text, Lexical, Masked ITN) should be used for entity detection. The details of the entities detected - like the offset, length and the text itself - will correspond to the text type selected here.",
           "$ref": "#/definitions/TranscriptContentType"
+        },
+        "redactionCharacter": {
+          "type": "string",
+          "description": "Optional parameter to use a Custom Character to be used for redaction in Conversational PII responses. Default character will be * as before.",
```

Comment:
nit: You can remove "as before". Also, don't need to have Custom Character be capitalized